nss-day-cohort-48/rare-client-orchesta-outlaws
# Added styling to navbar

Only made changes to css files

## To Test
`git fetch --all`, `git checkout al-navbar-style`
login to the app and check whether the new styling added to the navbar matches what is laid out in the wireframe.
